scipy/stats/tests/test_morestats.py

```diff
@@ -2655,26 +2664,29 @@ def test_nan_raise(self, test_func, x):
     def test_bad_nan_policy(self, test_func, x):
         assert_raises(ValueError, test_func, x, high=360, nan_policy='foobar')
 
+    @skip_xp_invalid_arg
```

Comment:
I think this is a valid use of `xp_invalid_arg`? Scalars are not arrays, so they will be rejected with array API turned on.